### PR TITLE
Change all CANSparkMax to TalonFX

### LINKED ISSUE #87: Swap NEOs For Falcons
Replace all `CANSparkMAX` objects with `TalonFX` objects and modify subsytems accordingly

### PR DESCRIPTION
**Pull request by MatthewZ0823**
Fix #87 

Change all of the CANSparkMax objects into TalonFX objects and made changes accordingly

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/Revolver.java [hunk 1]
```diff
@@ -19,18 +19,18 @@
 
 public class Revolver extends SubsystemBase {
 
-  private final CANSparkMax m_revolver;
+  private final TalonFX m_revolver;
```
Comment (Jacob4649) [inline src/main/java/frc/robot/subsystems/Revolver.java]: Change this to `WPI_TalonFX` to allow access to WPILib `SpeedController` methods

File: src/main/java/frc/robot/subsystems/Revolver.java [hunk 2]
```diff
@@ -42,7 +42,8 @@ public Revolver() {
    * @param speed the voltage to apply to the motor as a percentage from -1 to 1
    */
   public void set(double speed) {
-    m_revolver.set(speed);
+    m_revolver.set(ControlMode.PercentOutput, speed);
+    m_setSpeed = speed;
```
Comment (Jacob4649) [inline src/main/java/frc/robot/subsystems/Revolver.java]: This is not necessary if you switch `m_revolver` to a `WPI_TalonFX`

File: src/main/java/frc/robot/subsystems/Revolver.java [hunk 3]
```diff
@@ -19,18 +19,18 @@
 
 public class Revolver extends SubsystemBase {
 
-  private final CANSparkMax m_revolver;
+  private final TalonFX m_revolver;
   private int m_drumReps = 0;
+  private double m_setSpeed;
```
Comment (Jacob4649) [inline src/main/java/frc/robot/subsystems/Revolver.java]: This is not necessary if you change `m_revolver` to a `WPI_TalonFX`

File: src/main/java/frc/robot/subsystems/Revolver.java [hunk 4]
```diff
@@ -58,7 +59,8 @@ public void resetUnjamTimer() {
 
   @Override
   public void periodic() {
-    if (m_revolver.get() > 0.0 && m_revolver.getEncoder().getVelocity() < Constants.Revolver.VELOCITY_THRESHOLD) {
+    if (m_setSpeed > 0.0
```
Comment (Jacob4649) [inline src/main/java/frc/robot/subsystems/Revolver.java]: Use `m_revolver.get()` once you have changed `m_revolver` to a `WPI_TalonFX`

**Review by Jacob4649 — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/subsystems/Revolver.java [hunk 5]
```diff
@@ -58,7 +59,8 @@ public void resetUnjamTimer() {
...
+        && (m_revolver.getSelectedSensorVelocity() / 2048d) * 10d * 60d < Constants.Revolver.VELOCITY_THRESHOLD) {
```
Comment (Jacob4649) [inline src/main/java/frc/robot/subsystems/Revolver.java]: Move `(m_revolver.getSelectedSensorVelocity() / 2048d) * 10d * 60d` to a public method called `getVelocity` or something like that and call it here

**Review by Jacob4649 — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/subsystems/Revolver.java (hunk 1 above)
Comment (MatthewZ0823) [reply src/main/java/frc/robot/subsystems/Revolver.java]: Resolved in commit 4b623d01

File: src/main/java/frc/robot/subsystems/Revolver.java (hunk 2 above)
Comment (MatthewZ0823) [reply src/main/java/frc/robot/subsystems/Revolver.java]: Resolved in commit 4b623d0

File: src/main/java/frc/robot/subsystems/Revolver.java (hunk 3 above)
Comment (MatthewZ0823) [reply src/main/java/frc/robot/subsystems/Revolver.java]: Resolved in commit 4b623d0

File: src/main/java/frc/robot/subsystems/Revolver.java (hunk 4 above)
Comment (MatthewZ0823) [reply src/main/java/frc/robot/subsystems/Revolver.java]: Resolved in commit 4b623d0

File: src/main/java/frc/robot/subsystems/Revolver.java (hunk 5 above)
Comment (MatthewZ0823) [reply src/main/java/frc/robot/subsystems/Revolver.java]: Resolved in commit 4b623d0

File: src/main/java/frc/robot/subsystems/Revolver.java [hunk 6]
```diff
@@ -56,9 +55,13 @@ public void resetUnjamTimer() {
     m_timer.reset();
   }
```
Comment (Jacob4649) [inline src/main/java/frc/robot/subsystems/Revolver.java:60]: Add javadocs here describing what this method is, and what it returns (i.e. @returns the speed of the revolver in RPM)

**Review by Jacob4649 — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/subsystems/Revolver.java (hunk 6 above)
Comment (MatthewZ0823) [reply src/main/java/frc/robot/subsystems/Revolver.java:60]: Resolved in commit cceed37

Comment (MatthewZ0823) [reply src/main/java/frc/robot/subsystems/Revolver.java:60]: Resolved in commit dd412b9

**Review by Jacob4649 — APPROVED** [review]

**Pull request merged**